README.md

```diff
@@ -1,9 +1,4 @@
-## sandboxedFs
-
-[![npm version](https://badge.fury.io/js/sandboxed-fs.svg)](https://badge.fury.io/js/sandboxed-fs)
-[![Build Status](https://travis-ci.org/metarhia/sandboxed-fs.svg?branch=master)](https://travis-ci.org/metarhia/sandboxed-fs)
-[![NPM Downloads/Month](https://img.shields.io/npm/dm/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs)
-[![NPM Downloads](https://img.shields.io/npm/dt/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs)
+## sandboxed-fs [![npm version](https://img.shields.io/npm/v/sandboxed-fs.svg?style=flat)](https://www.npmjs.com/package/sandboxed-fs) [![Build Status](https://travis-ci.org/metarhia/sandboxed-fs.svg?branch=master)](https://travis-ci.org/metarhia/sandboxed-fs) [![NPM Downloads/Month](https://img.shields.io/npm/dm/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs) [![NPM Downloads](https://img.shields.io/npm/dt/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs) [![license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/metarhia/sandboxed-fs/blob/master/LICENSE)
 
 sandboxedFs is a sandboxed wrapper for Node.js file system module implementing
```

Comment:
We need to change the name here as well.
```suggestion
sandboxed-fs is a sandboxed wrapper for Node.js file system module implementing
```

Reply:
Fixed